languages/en/distance.js

```diff
@@ -0,0 +1,33 @@
+var lev = require('levenshtein');
+var tokenize = require('../../lib/tokenize');
+
+/**
+ * Finds the closest match between a statement and a body of words using
+ * Levenshtein Distance
```

Comment:
Levenshtein Distance is a great performant strategy, but if we are going to make this optional anyway we might want to discuss alternative edit distance algorithms (e.g. Myers Diff Algorithm).

PDF:
[myers.pdf](https://github.com/thisandagain/sentiment/files/2115441/myers.pdf)